docs/variables.md

```diff
@@ -184,6 +184,18 @@ Only available for `successCondition`
 | `response.body` | Response body (`string`) |
 | `response.headers` | Response headers (`map[string][]string`) |
 
+### CronWorkflow Templates
+
+> v3.6 and after, note that the the variables here can be used with the full power of expr-lang expression.
+
+| Variable | Description|
+|----------|------------|
+| `cronworkflows.name` | Name of the CronWorkflow method (`string`) |
+| `cronworkflows.numActive` | Number of running CronWorkflows (`int`) |
```

Comment:
I think you meant child Workflows
```suggestion
| `cronworkflow.numActive` | Number of running child Workflows (`int`) |
```

Reply:
I'm also not really sure we should expose this, as it's accuracy will fluctuate and so it's not that reliable

Reply:
Also while the `status` does call it "Active", that's not a state we use externally (it also might be a legacy naming thing). "Running" is the term used, and I would recommend that here too

Reply:
Alright, I am happy to remove this.